python/nano/scripts/bigdl-nano-init

```diff
@@ -215,11 +215,20 @@ if [ ! -z "${DISABLE_TCMALLOC_VAR:-}" ]; then
 	export LD_PRELOAD=`echo $LD_PRELOAD | sed "s/\s.*libtcmalloc\.so//g" | sed "s/.*libtcmalloc\.so\s*//g"`
 fi
 
-if [ -z "${LD_PRELOAD:-}" ]; then
-	unset LD_PRELOAD
+if [ -z "${CONDA_DEFAULT_ENV:-}" ]; then
+	echo "Not in a conda env"
+else
+    if [ -f $CONDA_PREFIX/etc/conda/activate.d/nano_vars.sh ];then
+        echo "nano_vars.sh already exists"
+    else
+        echo "Setting environment variables in current conda env"
+        mkdir -p $CONDA_PREFIX/etc/conda/activate.d
+        mkdir -p $CONDA_PREFIX/etc/conda/deactivate.d
+        echo "source bigdl-nano-init" > $CONDA_PREFIX/etc/conda/activate.d/nano_vars.sh
+		echo "source ${CONDA_PREFIX}/bin/bigdl-nano-unset-env" > $CONDA_PREFIX/etc/conda/deactivate.d/nano_vars.sh
```

Comment:
fix this indentation?

Reply:
Yes, it seems scripts in the deactivate.d folder run after exit the environment, so I add path prefix of `bigdl-nano-unset-env`.